src/components/CardFilm.js

```diff
@@ -111,28 +125,71 @@ class Item extends Component {
         this.setState({ dotsIsVisible: "no-show" });
     }
 
+    showModal() {
+        this.hideDots();
+        this.setState({ showModalWindow: true });
+    }
+
+    hideModal() {
+        this.setState({ showModalWindow: false });
+    }
+
+    handleClickTag(e) {
+        this.hideDots();
+        this.hideModal();
+
+        switch (e.target.innerHTML) {
+            case 'Edit':
+                this.props.dispatch(showEditPage(true));
+                break;
+            case 'Delete':
+                this.props.dispatch(showDeletePage(true));
+                break;
+            default:
+                break;
+         }
+    }
+
     render() {
         return (
             <StyledCard>
-                <MoreVertIcon className={this.state.dotsIsVisible}/>
+                <MoreVertIcon className={this.state.dotsIsVisible}
+                               onMouseEnter={this.showDots}
+                               onClick={this.showModal} />
+
+                <ModalWindow show={this.state.showModalWindow} handleClose={this.hideModal} onMouseOut={this.hideModal}>
+                    <p onClick={this.handleClickTag}>Edit</p>
+                    <p onClick={this.handleClickTag}>Delete</p>
+                </ModalWindow>
+
                 <Card.Img variant="top"
                           src={this.props.info.poster_path}
                           onMouseEnter={this.showDots}
-                          onMouseLeave={this.hideDots}
-                />
+                          onMouseLeave={this.hideDots} />
+
                 <Card.Body>
                     <StyledCartTitle>
-                        <Link to={{pathname: `/movies/${this.props.info.id}`}} onClick={e => this.handleRequests(e, this.props.info.genres)}>{this.props.info.title}</Link>
+                        <Link to={{pathname: `/movies/${this.props.info.id}`}}
+                              onClick={e => this.handleRequests(e, this.props.info.genres)}>
```

Comment:
```suggestion
                              onClick={this.handleRequests}>
```

You could get `this.props.info.genres` right inside the callback